src/org/loklak/objects/Timeline2.java

```diff
@@ -0,0 +1,457 @@
+/**
```

Comment:
this is Timeline class , just that, it iterates `Post` objects instead of `MessageEntry` objects.